.github/workflows/ci.yaml

```diff
@@ -11,10 +11,17 @@ on:
     branches:
       - "master"
 
+
 concurrency:
-  group: "${{ github.ref }}-${{ github.head_ref }}"
+  group: "${{ github.ref }}-${{ github.head_ref }}-${{ github.workflow }}"
   cancel-in-progress: true
 
+
+defaults:
```

Comment:
This is more of a suggested change (the old way isn't wrong, I just see this as cleaner but I'm happy to revert)

Reply:
nicer, thanks!